packages/website/ts/pages/credits.tsx

```diff
@@ -0,0 +1,121 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import { Banner } from 'ts/components/banner';
+import { Button } from 'ts/components/button';
+import { CenteredDefinition } from 'ts/components/centeredDefinition';
+import { Hero } from 'ts/components/hero';
+import { ModalContact, ModalContactType } from 'ts/components/modals/modal_contact';
+import { FlexWrap, Section } from 'ts/components/newLayout';
+import { SiteWrap } from 'ts/components/siteWrap';
+import { Heading } from 'ts/components/text';
+
+export interface NextCreditsProps {}
+
+export class NextCredits extends React.Component<NextCreditsProps> {
```

Comment:
No need to name things "Next". That is an artifact from the redesign (things were namespaced because we had two versions of the site at once).